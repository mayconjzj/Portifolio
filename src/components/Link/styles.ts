import tw from 'tailwind-styled-components';

export const Link = tw.a`
  text-blue-500
`;
