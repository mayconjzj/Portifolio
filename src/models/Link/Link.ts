export type LinkProps = {
  children: React.ReactNode;
  href: string;
};
