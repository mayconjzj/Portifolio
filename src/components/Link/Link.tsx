import * as S from './styles';

export const Link = ({
  children,
  href,
  ...props
}: {
  children: React.ReactNode;
  href: string;
}) => {
  return (
    <S.Link href={href} {...props}>
      {children}
    </S.Link>
  );
};
