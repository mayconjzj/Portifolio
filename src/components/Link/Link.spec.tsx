import { render } from '@testing-library/react';

import { Link } from './Link';

describe('Link', () => {
  it('should render successfully', () => {
    render(<Link href="/">Link</Link>);

    expect(true).toBeTruthy();
  });
});
