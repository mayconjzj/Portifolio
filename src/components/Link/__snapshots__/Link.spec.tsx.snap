// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link should render the link 1`] = `
<div>
  <a
    class="hover:border-b-2 hover:border-white cursor-pointer py-[2px] font-medium text-white transition-colors delay-150"
    isactive="false"
  >
    Next test
  </a>
</div>
`;
